src/library_gl.js

```diff
@@ -1400,6 +1411,97 @@ var LibraryGL = {
     GLctx.bufferSubData(target, offset, HEAPU8.subarray(data, data+size));
   },
 
+  // Queries EXT
```

Comment:
Yeah I duplicated it because I am not sure if we can overwrite the regular one with the extension one for WebGL2 and expect it to work for both regular and timer queries.